package.json

```diff
@@ -34,16 +34,19 @@
     "build": "npm run build:browser && npm run build:electron",
     "build:browser": "vite build",
     "build:browser:url": "vite build --base $PUBLIC_URL",
+    "build:plugin": "vite build --base /inspector --outDir ../../plugins/dist-browser",
```

Comment:
should it be a part of the common `build` script?

Reply:
I'd like to include this in `build` after completing npm publish work as well, so yes but not yet in this PR.